docs/getting-started/understanding-config-files.md

```diff
@@ -66,20 +68,46 @@ A minimal version of this file looks like this:
       "advertise": false
     }
   ],
-  "useLedgerAfterSlot": 0
+  "useLedgerAfterSlot": 0,
+  "bootstrapPeers": [
+    {
+      "address": "z.z.z.z",
+      "port": 3003
+    }
+  ]
 }
 ```
 
-* The main difference between `LocalRoots` and `PublicRoots` is the ability to specify various groups with varying valencies in the former. This can be valuable for informing your node about different targets within a group to achieve. `LocalRoots` is designed for peers that the node should always keep as hot, such as its own block producer. On the other hand, `PublicRoots` serves as a source of fallback peers, to be used if peers from the ledger (when `useLedgerAfterSlot` is disabled or unavailable) cannot be accessed.
-
-* This means that your node will initiate contact with the node at IP `x.x.x.x` on `port 3001` and resolve the DNS domain `y.y.y.y` (provided it exists). It will then make efforts to establish a connection with at least one of the resolved IPs.
-
-* `valency` (or `hotValency`) tells the node the number of connections it should attempt to select from the specified group. When a DNS address is provided, valency determines the count of resolved IP addresses for which the node should maintain an active (hot) connection.
-
-- `warmValency` is an optional field, similar to `valency` or `hotValency`, that informs the node about the number of peers it should maintain as established (warm). This field is optional and defaults to the value set in the `valency` or `hotValency` field. If a value is specified for `warmValency`, it should be greater than or equal to the one defined in `valency` or `hotValency`; otherwise, `valency` or `hotValency` will be adjusted to match this value. We recommend users set the `warmValency` value to `hotValency` + 1 to ensure at least one backup peer is available to be promoted to a hot connection in case of unexpected events.
+* The main difference between `LocalRoots` and `PublicRoots` is the ability to
+  specify various groups with varying valencies in the former. This can be
+  valuable for informing your node about different targets within a group to
+  achieve. `LocalRoots` is designed for peers that the node should always keep
+  as hot, such as its own block producer. On the other hand, `PublicRoots`
+  serves as a source of fallback peers, to be used if peers from the ledger
+  (when `useLedgerAfterSlot` is disabled or unavailable) cannot be accessed.
+
+* This means that your node will initiate contact with the node at IP
+  `x.x.x.x` on `port 3001` and resolve the DNS domain `y.y.y.y` (provided it
+  exists). It will then make efforts to establish a connection with at least
+  one of the resolved IPs.
+
+* `valency` (or `hotValency`) tells the node the number of connections it
+  should attempt to select from the specified group. When a DNS address is
+  provided, valency determines the count of resolved IP addresses for which
+  the node should maintain an active (hot) connection.
```

Comment:
`valency` still works, it is backward compatible. So that's I left it there.

Reply:
That's fine, my suggestion changes the accents.  With current style it looks like `valency` is the preferred way, but in fact it's something we will remove at some point, because there's no reason to support multiple syntaxes.